mantis-control-plane/mantis-control-plane-client/src/main/java/io/mantisrx/server/master/client/MantisMasterClientApi.java

```diff
@@ -727,7 +736,7 @@ public Observable<JobSchedulingInfo> schedulingChanges(final String jobId) {
                                     ;
                         }))
                 .repeatWhen(repeatLogic)
-                .retryWhen(retryLogic)
+                .retryWhen(retryObject.getRetryLogic())
```

Comment:
This feels like a convoluted way of saying not to retry after a "not found" exception.

Reply:
exactly that's how this ConditionalRetry class is designed i think?